lib/senkyoshi/models/content.rb

```diff
@@ -30,12 +30,17 @@ class Content < Resource
       "Senkyoshi::Quiz" => "Quizzes::Quiz",
     }.freeze
 
-    attr_accessor(:title, :body, :id, :files, :url)
+    attr_accessor(:title, :body, :files, :url)
     attr_reader(:extendeddata)
 
+    def initialize(resource_id = nil)
+      super(resource_id)
+    end
```

Comment:
I didn't realize that it would automatically inherit the non-default constructor. It looks like it does though, and it can go away.